src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/model/OffenceSexualOrViolent.kt

```diff
@@ -17,12 +19,16 @@ data class OffenceSexualOrViolent(
       inSchedule15Part2: Boolean,
       domesticViolence: Boolean,
       useOffenceCodesForSexual: Boolean,
+      isNationalSecurity: Boolean,
+      isSexOffenceLegislation: Boolean,
       offenceCode: String,
     ): OffenceSexualOrViolentIndicator {
       return when {
-        domesticViolence -> SEXUAL
         useOffenceCodesForSexual && (inSchedule15Part2 || isOffenceCodePrefixedWithSX03orSX56(offenceCode)) -> SEXUAL
```

Comment:
The logic to determine SEXUAL is dispersed through various parts of the code, ideally we should have the logic in one place. Will raise a tech debt ticket to address this

Reply:
Thank you!